packages/design-system/src/components/Checkbox/Checkbox.tsx

```diff
@@ -52,7 +52,7 @@ const CheckboxEl = React.forwardRef<CheckboxElement, CheckboxElProps>(
 
     return (
       <CheckboxRoot ref={composedRefs} checked={checked} onCheckedChange={setChecked} {...props}>
-        <CheckboxIndicator style={{ display: 'inline-flex', pointerEvents: 'auto' }} forceMount>
+        <CheckboxIndicator style={{ display: 'inline-flex', pointerEvents: 'auto', justifyContent: 'center', alignItems: 'center' }} forceMount>
```

Comment:
Confused why we are applying all the styles here inline. Couldn't we apply them to the CheckBoxIndicator styled component on line 103?

Reply:
Hey Mark,

This is my other git acc, yes you are right I must have overlooked the styled component below.

Reply:
Are you making the change or do you want me to repush the changes?

Reply:
No worries! If you can make the changes and push that would be great!

You will also need to add a [changeset](https://github.com/strapi/design-system/pull/1821#issuecomment-2477400451). You can find out how to do it here in the [contributing docs](https://github.com/strapi/design-system/blob/main/CONTRIBUTING.md#pull-requests)

Reply:
Hey Mark I've updated the inline style and added it to the style component.
Also added the changelog